sdks/python/build.gradle

```diff
@@ -230,3 +230,12 @@ task postCommit() {
   dependsOn "hdfsIntegrationTest"
   dependsOn "postCommitVRTests"
 }
+
+task dependencyUpdates(dependsOn: ':dependencyUpdates') {
+  doLast {
+    exec {
+      executable 'sh'
```

Comment:
Can you explain why we need this in addition to the script run by the Jenkins job ?